ChangeLog.md

```diff
@@ -1,10 +1,129 @@
 ##2016.08.03 version 2.0.0
+* Removal of Force parameters that were obsoleted in the previous release
```

Comment:
First bullet:

This release contains breaking changes.  Please see [the migration guide](link to your gist) for change details and the impact on existing scripts.